### PR TITLE
fix broken links and add ::class usage sample

### PR DESCRIPTION
**Pull request by dadaxr**

### CODE REVIEW

File: docs/extensions/storage/entity-manager.md [hunk 1]
```diff
@@ -34,13 +34,14 @@ relevant repository class.
 ### getRepository($entity)
 
 ```
+$repo = $app['storage']->getRepository(Bolt\Storage\Entity\Users::class);
```
Comment (PhillippOhlandt) [inline docs/extensions/storage/entity-manager.md]: Isn't it `\Bolt\Storage\Entity\Users::class`?

File: docs/extensions/storage/entity-manager.md [hunk 2]
```diff
@@ -34,8 +34,8 @@ relevant repository class.
 ### getRepository($entity)
 
 ```
-$repo = $app['storage']->getRepository(Bolt\Storage\Entity\Users::class);
-$repo = $app['storage']->getRepository('Bolt\Storage\Entity\Users');
+$repo = $app['storage']->getRepository(\Bolt\Storage\Entity\Users::class);
+$repo = $app['storage']->getRepository('\Bolt\Storage\Entity\Users');
```
Comment (PhillippOhlandt) [inline docs/extensions/storage/entity-manager.md]: This was fine before, I think.

Comment (PhillippOhlandt) [inline docs/extensions/storage/entity-manager.md]: The second one, without the `:class`

Comment (GawainLynch) [inline docs/extensions/storage/entity-manager.md]: No, `::class` is correct

Comment (GawainLynch) [inline docs/extensions/storage/entity-manager.md]: You are also correct … I can't read … must … bed

**Comment by GawainLynch** [comment]
Merged into `release/3.0` via 29af786c72318b33fbee5fbd35bea2d68ef7b80d

**Pull request closed without merging**